### PR TITLE
Issue with nupi

### PR DESCRIPTION
**Pull request by tallamohan**
Fixing the issue with NUPI in clearscape database by changing the dbscontrol options to:
m g 53=P

### CODE REVIEW

File: .github/workflows/scripts/createTestEnvironments.sh [hunk 1]
```diff
@@ -37,7 +37,8 @@ createRegularEnv() {
   --data-raw "{
       \"name\": \"$CSAE_ENV_NAME\",
       \"region\": \"us-central\",
-      \"password\": \"$CSAE_ENV_PASSWORD\"
+      \"password\": \"$CSAE_ENV_PASSWORD\",
+      \"startupScript\": \"#!/bin/bash\n\nfunction withRetry {\n  local RETRIES=\$1; shift 1\n  local SLEEP=\$1; shift 1\n  for i in \$(seq 1 \$RETRIES)\n  do\n    echo \\\"Attempt \$i: Running command \$@\\\"\n    \$@ && s=0 && break || s=\$? && sleep \$SLEEP\n  done\n  return \$s\n}\n\nfunction ensureVantageIsUp {\n  pdestate -a\n  pdestate -a | grep \\\"DBS state is [45]\\\"\n}\n\ndbscontrol << EOF\nm g 53=P\nW\nEOF\ntpareset -y changing dbscontrol\nwithRetry 40 2 ensureVantageIsUp\"
```
Comment (adamtwo) [inline .github/workflows/scripts/createTestEnvironments.sh:41]: like we discussed in the email, changes to `g 53` do not require a restart. You can simplify this startup script to something like this (have not tested, sharing this as an example):

```
\"startupScript\": \"#!/bin/bash\n\ndbscontrol << EOF\nm g 53=P\nW\nEOF\n"
```

File: .github/workflows/scripts/createTestEnvironments.sh [hunk 2]
```diff
@@ -57,7 +58,7 @@ createEnvWithQVCI() {
   echo "teradata-server-name-qvci=$TERADATA_SERVER_NAME" >> $GITHUB_OUTPUT
 }
 
-background createRegularEnv
-background createEnvWithQVCI
+createRegularEnv
```
Comment (adamtwo) [inline .github/workflows/scripts/createTestEnvironments.sh:61]: `background` command puts the function invocation in a separate process. This allows `createReuglarEnv` and `createEnvWithQVCI` to proceed in parallel. When you remove `background`, you change it to run sequentially, thus making it run longer. I'm not sure what we gain by running it in sequence?

**Review by adamtwo — CHANGES_REQUESTED** [review]

File: .github/workflows/scripts/createTestEnvironments.sh (hunk 2 above)
Comment (tallamohan) [reply .github/workflows/scripts/createTestEnvironments.sh:61]: Correct! Adding `background ` back.

**Review by VarunSharma15 — APPROVED** [review]
Looks Good To Me

**Review by tallamohan — COMMENTED** [review]
Addressed the review comments

**Pull request merged**